arch/risc-v/src/mpfs/chip.h

```diff
@@ -29,4 +29,33 @@
 
 #include "mpfs_memorymap.h"
 
+#include "riscv_percpu.h"
+
+/****************************************************************************
+ * Macro Definitions
+ ****************************************************************************/
+
+#ifdef __ASSEMBLY__
+
+/****************************************************************************
+ * Name: setintstack
+ *
+ * Description:
+ *   Set the current stack pointer to the  "top" the correct interrupt stack
+ *   for the current CPU.
+ *
+ ****************************************************************************/
+
+#if CONFIG_ARCH_INTERRUPTSTACK > 15
+.macro  setintstack tmp0, tmp1
+#ifndef CONFIG_ARCH_USE_S_MODE
```

Comment:
Looks like the definition of the macro needs to be inside the second `#ifdef`, otherwise we are ending up with two definitions.

Reply:
Yes this is wrong, I'll fix on monday

Reply:
Done